docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,17 +3,179 @@ title: Database Access AWS IAM Reference
 description: AWS IAM policies for Teleport database access.
 ---
 
+## Auto-discovery
 Teleport automatically discovers and configures IAM for RDS and Redshift given
 proper IAM permissions as described in the [AWS RDS & Aurora
 guide](../guides/rds.mdx) and the [AWS Redshift
 guide](../guides/postgres-redshift.mdx).
 
+The Teleport Database Service requires a set of AWS IAM permissions to:
+- update database configurations
+- update its identity with connection-related permissions
+
+You can generate and manage the permissions with the [`teleport db configure
+bootstrap`](../../database-access/reference/cli.mdx#teleport-db-configure-bootstrap)
+command. For example, the following command would generate and print the
+IAM policies:
+
+```code
```

Comment:
I think this approach works. The example command is brief enough to indicate what the configurator feature does, and there's a link to the CLI reference.